assets/js/components/setup/ModuleSetup.js

```diff
@@ -101,15 +99,7 @@ export default function ModuleSetup( { moduleSlug } ) {
 
 	return (
 		<Fragment>
-			<Header>
-				<HelpMenu>
```

Comment:
@hussain-t same here – the `HelpMenu` should still be rendered, only without the added child.